decision_service/dsclientlib/ds_driver_impl.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include "logger/ds_logger.h"
+
+namespace decision_service
+{
+  class ranking_response;
+  class api_status;
+
+  class driver_impl {
```

Comment:
Why is driver_impl the right name?  It sounds quite generic.

Reply:
Perhaps the 'driver' part is the issue.   A library that has 'driver' as a primary object seems quite generic.  Is there a better descriptor for what 'driver' is supposed to do?

Reply:
A couple of suggestions:
a) decision_service
b) ds_client